app/assets/javascripts/application.js

```diff
@@ -12,5 +12,5 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require bootstrap-sprockets
 //= require jasny-bootstrap
-//= require_tree .
```

Comment:
Embedded JS files may be different from Madeline System JS files. So not including whole tree anymore in application template.